### PR TITLE
wordpress: 6.0.3 -> 6.1.1

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by risicle** [comment]
Regarding the backport label - is this a security fix?

Is the 6.0 branch being abandoned?

**Comment by onny** [comment]
> Regarding the backport label - is this a security fix?
> 
> Is the 6.0 branch being abandoned?

The 6.0 branch is still supported and currently at 6.0.3. I'm not sure if 6.1 introduced any security fixes but you're right, we shouldn't backport it and stay with 6.0 on NixOS 22.11

**Comment by SuperSandro2000** [comment]
Did you test this on a deployment?

**Comment by urandom2** [comment]
I have tested it on a localhost deployment and can confirm all tested features work

File: pkgs/servers/web-apps/wordpress/default.nix [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "wordpress";
-  version = "6.0.3";
+  version = "6.1.1";
 
   src = fetchurl {
     url = "https://wordpress.org/${pname}-${version}.tar.gz";
-    sha256 = "sha256-eSi0qwzXoJ1wYUzi34s7QbBbwRm2hfXoyGFivhPBq5o=";
+    sha256 = "sha256-IR6FSmm3Pd8cCHNQTH1oIaLYsEP1obVjr0bDJkD7H60=";
```
Comment (urandom2) [inline pkgs/servers/web-apps/wordpress/default.nix:9]: nit
```suggestion
    hash = "sha256-IR6FSmm3Pd8cCHNQTH1oIaLYsEP1obVjr0bDJkD7H60=";
```

**Comment by urandom2** [comment]
> > Regarding the backport label - is this a security fix?
> > Is the 6.0 branch being abandoned?
> 
> The 6.0 branch is still supported and currently at 6.0.3. I'm not sure if 6.1 introduced any security fixes but you're right, we shouldn't backport it and stay with 6.0 on NixOS 22.11

what is the correct way to get this visible in stable? currently wordpress gives an upgrade banner on a bunch of pages, and silencing that would be very nice. does it make sense to do a `pkgs.wordpress_6_1`?

**Comment by onny** [comment]
Result of `nixpkgs-review pr 208529` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 test built:</summary>
  <ul>
    <li>nixosTests.wordpress</li>
  </ul>
</details>

**Pull request merged**